scipy/linalg/flapack_other.pyf.src

```diff
@@ -1854,6 +1854,21 @@ subroutine <prefix>lartg(f,g,cs,sn,r)
 end subroutine <prefix>lartg
 
 subroutine <prefix2c>rot(n,x,offx,incx,y,offy,incy,c,s,lx,ly)
+    ! ?GEJSV computes the singular value decomposition (SVD) of a real M-by-N
```

Comment:
I think this doesn't belong here.

Reply:
..... I think you're right. I'll fix it in the next set of commits.